project.conf

```diff
@@ -0,0 +1,306 @@
+        else autoreconf -ivf
+        fi
+      # the directory where config.sub and config.guess are stored
+      conf-aux: .
+
+  cmake:
+    variables:
+      cmake-global: -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      generator: Ninja
+  distutils:
+    variables:
+      prefix: /usr/local
+      python-install: '%{python} setup.py install --root "%{install-root}"'
+      fix-pyc-timestamps: ''
+    config:
+      install-commands:
+        (>):
+          # There's no way to get the setup.py to install this in the right place.
+        - |
+          if [ -d '%{install-root}/usr/lib/pkgconfig/' ]; then
+            mkdir -p %{install-root}/usr/%{lib}/
+            mv -f %{install-root}/usr/lib/pkgconfig/ %{install-root}/usr/%{lib}/
+          fi
+  meson:
+    variables:
+      meson-global: --buildtype=plain --auto-features=enabled
+  pip:
+    variables:
+      pip: pip3
+      fix-pyc-timestamps: "true"
+    config:
+      install-commands:
+        - |
+          %{pip} install --no-build-isolation --no-deps --root=%{install-root} --prefix=%{prefix} .
+  filter:
+    config:
+      include-orphans: true
+
+sources:
+  git_tag:
+    config:
+      checkout-submodules: false
+      track-tags: false
+
+split-rules:
+  devel:
+    (>):
+    - '%{libdir}/cmake'
+    - '%{libdir}/cmake/**'
+    - '%{libdir}/cmake/**'
+    - '%{datadir}/gir-1.0/**'
+    - '%{datadir}/vala*/vapi'
+  vm:
+  - '%{datadir}/dbus-1/**'
+
+# Define some behavior for `bst shell`
+#
+shell:
+
+  # Default command for `bst shell`, we prefer bash, and disable
+  # parsing of profile and rc files so that the prompt BuildStream
+  # sets is not overwritten.
+  command: ['bash', '--noprofile', '--norc', '-i']
+
+  # Some environment variables to inherit from the host environment
+  environment:
+    LANG: '$LANG'
+    DISPLAY: '$DISPLAY'
+    DBUS_SESSION_BUS_ADDRESS: '$DBUS_SESSION_BUS_ADDRESS'
+    XDG_RUNTIME_DIR: '$XDG_RUNTIME_DIR'
+
+    # Inform applications which use pulseaudio of the server socket
+    PULSE_SERVER: 'unix:${XDG_RUNTIME_DIR}/pulse/native'
+
+  # Some things to mount into the sandbox
+  host-files:
+  # Understand user inherited uid/gid
+  - '/etc/passwd'
+  - '/etc/group'
+  # Allow network resolution
+  - '/etc/resolv.conf'
+
+  # Allow access to plausible video devices,
+  # declare these optional to avoid meaningless warnings
+  - path: '/dev/dri'
+    optional: true
+  - path: '/dev/mali'
+    optional: true
+  - path: '/dev/mali0'
+    optional: true
+  - path: '/dev/umplock'
+    optional: true
+  - path: '/dev/nvidiactl'
+    optional: true
+  - path: '/dev/nvidia0'
+    optional: true
+  - path: '/dev/nvidia-modeset'
+    optional: true
+
+  # Allow access to sysfs, needed for local device discovery
+  - '/sys'
+
+  # Allow access to the user runtime directory, this
+  # will include the pulseaudio socket along with some
+  # other things.
+  - '${XDG_RUNTIME_DIR}'
+
+variables:
+  sdk-branch: "6"
```

Comment:
Should this be `master` on master? Currently when you build master it creates `runtime/com.endlessm.apps.Sdk/*/master`.

Reply:
Okay, changed this to master, and changed a bunch of other elements to track the master branches of their respective repos. When this is merged we can add a commit in the sdk-6 branch that tracks sdk-6 branches instead.